Content.Shared/Physics/CollisionGroup.cs

```diff
@@ -14,14 +14,15 @@ namespace Content.Shared.Physics;
 public enum CollisionGroup
 {
     None               = 0,
-    Opaque             = 1 << 0, // 1 Blocks light, can be hit by lasers
-    Impassable         = 1 << 1, // 2 Walls, objects impassable by any means
-    MidImpassable      = 1 << 2, // 4 Mobs, players, crabs, etc
-    HighImpassable     = 1 << 3, // 8 Things on top of tables and things that block tall/large mobs.
-    LowImpassable      = 1 << 4, // 16 For things that can fit under a table or squeeze under an airlock
-    GhostImpassable    = 1 << 5, // 32 Things impassible by ghosts/observers, ie blessed tiles or forcefields
-    BulletImpassable   = 1 << 6, // 64 Can be hit by bullets
-    InteractImpassable = 1 << 7, // 128 Blocks interaction/InRangeUnobstructed
+    XrayImpassable     = 1 << 0, // 1 Ignores walls, can only hit mobs
```

Comment:
This should just be a mob layer, don't hardcode an entire layer for 1 interaction.